bokehjs/gulp/tasks/generate_defaults.py

```diff
@@ -104,8 +104,5 @@ def output_defaults_module(filename, defaults):
 
 output_defaults_module(filename = os.path.join(dest_dir, 'common/defaults.coffee'),
                        defaults = all_json)
-output_defaults_module(filename = os.path.join(dest_dir, 'widget/defaults.coffee'),
+output_defaults_module(filename = os.path.join(dest_dir, 'models/widgets/defaults.coffee'),
```

Comment:
On the other hand auto-generated defaults could be put in build (or maybe in test), because we only use them in tests. However, using `build/` directory, will require changes to module resolution code.